storage/connectors/sstable/src/main/java/feast/storage/connectors/sstable/retriever/SSTableOnlineRetriever.java

```diff
@@ -137,4 +141,22 @@ default String valueToString(ValueProto.Value v) {
         .distinct()
         .collect(Collectors.toList());
   }
+
+  /**
+   * Trims long SSTable table names and appends hash suffix for uniqueness.
+   *
+   * @param expr Original SSTable table name
+   * @param maxLength Maximum length allowed for SSTable
+   * @return Hashed suffix SSTable table name
+   */
+  default String trimAndHash(String expr, int maxLength) {
```

Comment:
The maximum possible length of the hash is 10 characters. Since the max prefix length is 40, the maximum possible length of the generated string would be 50. Wouldn't this exceed the 48 characters limit of cassandra?

Reply:
why this code is duplicated?